spec/plugins/partner_sku_reviews_spec.coffee

```diff
@@ -535,6 +540,9 @@ describe 'PartnerSkuReviews', ->
           it 'displays rating stars', ->
             expect(@subject.querySelector('.sa-star-rating').innerHTML.trim()).to.not.be.empty
 
+          it 'displays verification mark', ->
+            expect(@subject.querySelector('.sa-verification-mark').innerHTML.trim()).to.not.be.empty
```

Comment:
I'd suggest us to be a bit more specific here, with something among the lines of:

```suggestion
          context 'when the review is regarding a purchased product', ->
            beforeEach ->
              @subject = parent_doc.querySelectorAll('#\\@\\@flavor-product-reviews-extended .sa-reviews-list .sa-review')[0]

            it 'displays verification mark', ->
              expect(@subject.querySelector('.sa-verification-mark').innerHTML.trim()).to.not.be.empty
```

Reply:
Fixed it! Thanks!